packages/@aws-cdk/aws-ecs-patterns/lib/fargate/network-load-balanced-fargate-service.ts

```diff
@@ -123,6 +124,10 @@ export class NetworkLoadBalancedFargateService extends NetworkLoadBalancedServic
       throw new Error('You must specify one of: taskDefinition or image');
     }
 
+    // determine the ingress container port
+    const containerPort = props.taskImageOptions ? props.taskImageOptions.containerPort || 80 :
+      props.taskDefinition ? props.taskDefinition.defaultContainer!.ingressPort : 80;
```

Comment:
Please submit this change as a separate PR.

Reply:
OK. removed.